elog/elog.py

```diff
@@ -132,6 +145,8 @@ def __init__(self, instrument, station=None, user=None, pw=None,
         self.logbooks['experiment'] = exp_id
         register_elog(self, primary=primary)
 
+        self.enable_run_posts = enable_run_posts
```

Comment:
A comment on why this is unused here (and where it comes in) might be worthwhile